helm/sematic-server/templates/server-role.yaml

```diff
@@ -7,7 +7,7 @@ metadata:
 rules:
 - apiGroups: ["batch"]
   resources: ["jobs"]
-  verbs: ["get", "watch", "list", "create"]
+  verbs: ["get", "watch", "list", "create", "delete"]
```

Comment:
The Server that created a pod implicitly had "delete" privileges. This won't be true when scaling out, so explicitly adding here.